libwhisper/WhisperPeer.cpp

```diff
@@ -32,7 +31,11 @@ using namespace dev::shh;
 WhisperPeer::WhisperPeer(Session* _s, HostCapabilityFace* _h, unsigned _i, CapDesc const&): Capability(_s, _h, _i)
 {
 	RLPStream s;
-	sealAndSend(prep(s, StatusPacket, 1) << version());
+	//sealAndSend(prep(s, StatusPacket, 1) << version());
+	prep(s, StatusPacket, 2);
+	s << version();
+	s << host()->bloom();
```

Comment:
:+1: 
Note/todo: I'll help you test this. We might also remove the version() as it's now communicated in the handshake.